drivers/hmis/app/models/hmis/hud/validators/enrollment_validator.rb

```diff
@@ -34,23 +34,24 @@ def self.already_enrolled_full_message
     'Client has another enrollment in this project that conflicts with this entry date.'
   end
 
-  def self.find_conflict_severity(enrollment)
+  def self.find_conflicting_enrollment(enrollment)
```

Comment:
Is there test coverage for the changes here?

It looks like all the Enrollment creation tests are jammed into the submit_form_spec which is big https://github.com/greenriver/hmis-warehouse/blob/a65d9cddea0badf29b3fb1bc96ab025e4e18472c/drivers/hmis/spec/requests/hmis/submit_form_spec.rb#L468-L681

This seems like the right place to add assertions on the `data` that is returned:  https://github.com/greenriver/hmis-warehouse/blob/a65d9cddea0badf29b3fb1bc96ab025e4e18472c/drivers/hmis/spec/requests/hmis/submit_form_spec.rb#L540-L545